Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md

```diff
@@ -0,0 +1,54 @@
+---
+versionFrom: 8.0.0
```

Comment:
This applies to Umbraco 7 as well, right?
If it does, we should just use versionFrom: 7.0.0 - what do you think?

Reply:
Will use versionFrom 7.0.0 instead :)